benefits/static/css/styles.css

```diff
@@ -370,6 +370,14 @@ footer .footer-links li a:visited {
   line-height: 1;
 }
 
+#login .fallback-text.white-logo {
+  background-image: url("/static/img/login-gov-logo-rev.svg");
+}
+
+#login .fallback-text.color-logo {
+  background-image: url("/static/img/login-gov-logo.svg");
+}
```

Comment:
@angela-tran For when you start coding up these 3 - modal trigger link, modal title and modal title - you can use `#login .fallback-text.color-logo`. Currently `#login .fallback-text` has a set width and height, but you can edit the CSS to become more generic (remove the width/height from the main class) and add the width/height to a specific class, to be used in those 3 spots.
<img width="171" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/52d1698c-5e25-4281-81b2-66931c148945">
<img width="243" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/46a9a2b5-5ce3-409c-92ea-1338103fbb8b">
<img width="550" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/c053b515-dbdd-4c70-b98b-98ceb93e9dab">

Reply:
Thanks @machikoyasuda!